src/types/aws/ap/registration.ts

```diff
@@ -78,7 +79,7 @@ export type TDocumentation = {
   KycDonorsOnly: boolean;
   CashEligible: boolean;
   HqCountry: string;
-  EndowDesignation: string;
+  EndowDesignation: EndowDesignation | "";
```

Comment:
nit: there are a couple of places where either `EndowDesignation` or an empty string `""` is allowed as a value. It could make sense to add the empty string `""` to `EndowDesignation` and treat it as some kind of a "default" value.

Reply:
> nit: there are a couple of places where either `EndowDesignation` or an empty string `""` is allowed as a value. It could make sense to add the empty string `""` to `EndowDesignation` and treat it as some kind of a "default" value.
> 
> @SovereignAndrey can there be endowments for whom the `endowment designation` is empty?

@misicnenad @ap-justin All Charity Endowments would need to have some form of Endowment Designation. The AST Endowments would not have this today, though I would suspect they could get their own set of sub-designation at some point in the future.